mail_tracking/static/src/xml/mail_tracking.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<template>
+
+<t t-name="mail.tracking.status">
+    <t t-if="tracking[0] == 'unknown'">
+        <span class="mail_tracking_unknown">
+            <i class="fa fa-ban"></i>
+        </span>
+    </t>
+    <t t-if="tracking[0] == 'waiting'">
+        <span class="mail_tracking_waiting">
+            <i class="fa fa-clock-o"></i>
+        </span>
+    </t>
+    <t t-if="tracking[0] == 'error'">
+        <span class="mail_tracking_error">
+            <i class="fa fa-remove"></i>
+        </span>
+    </t>
+    <t t-if="tracking[0] == 'sent'">
+        <span class="mail_tracking_sent">
+            <i class="fa fa-check"></i>
+        </span>
+    </t>
+    <t t-if="tracking[0] == 'delivered'">
+        <span class="fa-stack mail_tracking_delivered">
+            <i class="fa fa-check fa-stack-1x" style="margin-left:1px"></i>
+            <i class="fa fa-check fa-inverse fa-stack-1x" style="margin-left:-2px;"></i>
+            <i class="fa fa-check fa-stack-1x" style="margin-left:-3px"></i>
+        </span>
+    </t>
+    <t t-if="tracking[0] == 'opened'">
+        <span class="fa-stack mail_tracking_opened">
+            <i class="fa fa-check fa-stack-1x" style="margin-left:1px"></i>
+            <i class="fa fa-check fa-inverse fa-stack-1x" style="margin-left:-2px;"></i>
+            <i class="fa fa-check fa-stack-1x" style="margin-left:-3px"></i>
+        </span>
+    </t>
+</t>
+
+<t t-extend="mail.thread.message">
+    <t t-jquery="span[t-attf-class='oe_partner_follower']" t-operation="append">
+        <t t-set='tracking' t-value='widget.partner_trackings[partner[0]]'/>
+        <t t-if="widget.options.show_link and tracking[1]">
```

Comment:
It's sad because this would be very useful, but nobody adds this "show_link" option in core odoo. IMHO it should just check if the user has rights and then show the link if so, or show it always and let Odoo show the access error, or the option be "hide_link", or default "show_link" to True.